docker-compose/database.properties

```diff
@@ -0,0 +1,6 @@
+driver=postgresql
+host=172.17.0.1
+database=directory
```

Comment:
remove the batabase property so that this file is generic and can be used by every service that require a postgre (it'll still be defined in the database.properties from the different repos)

Reply:
database property removed from file.